core/field_variable.js

```diff
@@ -171,13 +182,17 @@ Blockly.FieldVariable.dropdownCreate = function() {
   if (this.sourceBlock_) {
     workspace = this.sourceBlock_.workspace;
   }
+  var isBroadcastType = false;
   if (workspace) {
     var variableTypes = this.getVariableTypes_();
     var variableModelList = [];
     // Get a copy of the list, so that adding rename and new variable options
     // doesn't modify the workspace's list.
     for (var i = 0; i < variableTypes.length; i++) {
       var variableType = variableTypes[i];
+      if (variableType === 'broadcast_msg'){
```

Comment:
Make 'broadcast_msg' a constant, probably in constants.js.

Reply:
Should I also use == here instead of ===?

Reply:
Yes.